docs/src/main/asciidoc/security-oidc-bearer-token-authentication.adoc

```diff
@@ -1026,44 +1075,50 @@ Note that `@TestSecurity` `user` and `roles` attributes are available as `TokenI
+To see more details about token verification errors, enable `io.quarkus.oidc.runtime.OidcProvider` and `TRACE` level logging:
 
 [source, properties]
 ----
 quarkus.log.category."io.quarkus.oidc.runtime.OidcProvider".level=TRACE
 quarkus.log.category."io.quarkus.oidc.runtime.OidcProvider".min-level=TRACE
 ----
 
-Please enable `io.quarkus.oidc.runtime.OidcRecorder` `TRACE` level logging to see more details about the OidcProvider client initialization errors:
+To see more details about `OidcProvider` client initialization errors, enable `io.quarkus.oidc.runtime.OidcRecorder` and `TRACE` level logging as follows:
 
 [source, properties]
 ----
 quarkus.log.category."io.quarkus.oidc.runtime.OidcRecorder".level=TRACE
 quarkus.log.category."io.quarkus.oidc.runtime.OidcRecorder".min-level=TRACE
 ----
 
-=== External and Internal Access to OpenID Connect Provider
+=== External and internal access to OIDC providers
 
-Note that the OpenID Connect Provider externally accessible token and other endpoints may have different HTTP(S) URLs compared to the URLs auto-discovered or configured relative to `quarkus.oidc.auth-server-url` internal URL. For example, if your SPA acquires a token from an external token endpoint address and sends it to Quarkus as a bearer token then an issuer verification failure may be reported by the endpoint.
+The externally-accessible token of the OIDC provider and other endpoints might have different HTTP(S) URLs compared to the URLs that are auto-discovered or configured relative to the `quarkus.oidc.auth-server-url` internal URL.
+For example, suppose your SPA acquires a token from an external token endpoint address and sends it to Quarkus as a bearer token.
+In that case, the endpoint might report an issuer verification failure.
 
-In such cases, if you work with Keycloak then please start it with a `KEYCLOAK_FRONTEND_URL` system property set to the externally accessible base URL.
-If you work with other Openid Connect providers then please check your provider's documentation.
+In such cases, if you work with Keycloak, start it with the `KEYCLOAK_FRONTEND_URL` system property set to the externally-accessible base URL.
+If you work with other OIDC providers, refer to your provider's documentation.
 
-=== How to use 'client-id' property
+[[using-client-id]]
+=== Using the `client-id` property
 
-`quarkus.oidc.client-id` property identifies an OpenID Connect Client which requested the current bearer token. It can be an SPA application running in a browser or a Quarkus `web-app` confidential client application propagating the access token to the Quarkus `service` application.
+The `quarkus.oidc.client-id` property identifies the OIDC client that requested the current bearer token.
+The client requester can be an SPA application running in a browser or a Quarkus `web-app` confidential client application propagating the access token to the Quarkus `service` application.
```

Comment:
Thanks @MichalMaler, however, "an SPA" is correct here because "SPA" is pronounced as a single word that starts with a vowel sound. Therefore "an" is correct before this vowel sound.